openmaptiles/tileset.py

```diff
@@ -181,7 +196,49 @@ def description(self) -> str:
 
     @property
     def buffer_size(self) -> int:
-        return self.definition['layer']['buffer_size']
+        """Each layer must have a default buffer size, with either `buffer_size` or `min_buffer_size` or both.
+        If both are set, buffer_size must be >= min_buffer_size.
+        min_buffer_size is only used when there is a global buffer size override,
```

Comment:
My understanding is that `min_buffer_size` is also used with per-layer override.

```suggestion
        `min_buffer_size` is only used when there is a `buffer_size` override.
```